### PR TITLE
Don't fail GHES workflow if nothing changed

### PR DESCRIPTION
**Pull request by cdb**
Should fix the failure here: https://github.com/actions/starter-workflows/actions/runs/110902898

### CODE REVIEW

**Review by cschleiden — APPROVED** [review]
LGTM!

**Comment by awingfield916-tech** [comment]
Bug

File: .github/workflows/validate-data.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+name: Validate Data
+
 on:
```
Comment (awingfield916-tech) [inline .github/workflows/validate-data.yaml:3]: Bug

File: .github/workflows/validate-data.yaml [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
   push:
   pull_request:
```
Comment (awingfield916-tech) [inline .github/workflows/validate-data.yaml:5]: Bug

**Pull request merged**